libsolidity/formal/CHC.cpp

```diff
@@ -219,7 +219,10 @@ bool CHC::visit(FunctionDefinition const& _function)
 	if (!m_currentContract)
 		return false;
 
-	if (!_function.isImplemented())
+	if (
+		!_function.isImplemented() ||
+		abstractAsNondet(_function)
```

Comment:
What about in BMC?

Reply:
BMC unimplemented atm. It's not such a problem there because the analysis doesn't take the entire state into account anyway.